python/main.py

```diff
@@ -48,23 +91,27 @@ async def add_item(name: str = Form(...), category: str = Form(...), image: Uplo
     logger.info(f"Receive item, name: {name}, category: {category}")
     
     hashedImageName = await hashImage(image)
-    newItem = {
-        'name': name,
-        'category': category,
-        'image_filename': hashedImageName
-    }
-    allItems = getItems()
-    allItems["items"].append(newItem)
-
-    with open(fileName, 'w') as itemsFile:
-        json.dump(allItems, itemsFile)
+    postDbItem(name, category, hashedImageName)
 
     return {"message": f"item received with name: {name}, category: {category}, image: {hashedImageName}"}
 
 @app.get("/items")
 def get_items_reponse():
```

Comment:
nit: you might wanna change the function names. FastAPI has a thing called SwaggerUI where you can visually check and test your endpoints. The names of the functions serve as a description of the endpoints 👍

Ref:
https://fastapi.tiangolo.com/tutorial/first-steps/

Reply:
@walnut07 thank you for the comments! Changes made and merged!